modules/Gooey Example/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 posttype: "module"
 title: "Gooey Example"
```

Comment:
Should spaces be allowed in module names when used as a directory? Could that cause trouble if it ends up in a URL?

Reply:
It won't matter as the the url are generated via slugs that will put a hypen in space of space.